### PR TITLE
MariaDB Destination

### PR DESCRIPTION
**Pull request by wemrysi**
[ch11555]

### CODE REVIEW

File: core/src/main/scala/quasar/plugin/mariadb/DriverParameter.scala [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import monocle.Lens
+
+final case class DriverParameter(name: String, value: String) {
+  def forUrl: String = s"$name=$value"
+}
+
+object DriverParameter {
+  val NameValue: Regex = "^([^=]+)=(.*)$".r
+
+  object Optics {                                                                           val name: Lens[DriverParameter, String] =
```
Comment (cryogenian) [inline core/src/main/scala/quasar/plugin/mariadb/DriverParameter.scala]: :D

File: destination/src/main/scala/quasar/plugin/mariadb/destination/MariaDbDestination.scala [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+          CHAR,
+          VARBINARY,
+          BINARY,
+          BLOB,
+          MEDIUMBLOB,
+          LONGBLOB,
+          TINYTEXT,
+          TINYBLOB)
+
+      case _ => TypeCoercion.Unsatisfied(Nil, None)
```
Comment (cryogenian) [inline destination/src/main/scala/quasar/plugin/mariadb/destination/MariaDbDestination.scala:111]: Shouldn't offset date|time return `Unsatisfied(List(LocalDate|time)`?

**Review by cryogenian — CHANGES_REQUESTED** [review]
I think that `OffsetDatetime` and friends should return `Unsatisfied` with local options inside

**Comment by wemrysi** [comment]
@cryogenian good call on offering the alternatives, ready for another look.

**Review by cryogenian — APPROVED** [review]
Thanks :+1:

**Pull request merged**